### PR TITLE
Added getPlaylistID

### PR DESCRIPTION
**Pull request by Dmitry221060**
Solution for #33

### CODE REVIEW

**Comment by TimeForANinja** [comment]
first of all: thanks for the pr.
I'm fine with exporting the getPlaylistID function, but i'd prefer only returning a promise when no callback is provided, like here https://github.com/TimeForANinja/node-ytpl/blob/master/lib/main.js#L18 - that way we don't have to change all the internal usages and support quite a lot of older nodejs versions
Would also be cool if you add the entry to the README

**Comment by Dmitry221060** [comment]
That sounds fair, but I have no experience in testing, so here are a few questions:
1. Should we add tests to `main-test` if we just export the method from `util`? If so, should we use the same tests in both of them or move the tests from `util-test` to `main-test`?
2. Should we use only callbacks/Promises in the tests, or are there any rules for this?
Thanks for your reply!

File: lib/util.js [hunk 1]
```diff
@@ -74,6 +74,15 @@ const PLAYLIST_REGEX = exports.PLAYLIST_REGEX = /^(PL|UU|LL|RD)[a-zA-Z0-9-_]{16,
 const ALBUM_REGEX = exports.ALBUM_REGEX = /^OLAK5uy_[a-zA-Z0-9-_]{33}$/;
 const CHANNEL_REGEX = exports.CHANNEL_REGEX = /^UC[a-zA-Z0-9-_]{22,32}$/;
 exports.getPlaylistId = (link, callback) => {
+  // Return promise if no callback is defined
+  if (!callback) {
+    return new Promise((resolve, reject) => {
+      exports.getPlaylistId(link, (err, info) => { // eslint-disable-line consistent-return
+        if (err) reject(err);
```
Comment (TimeForANinja) [inline lib/util.js]: i'd add another return here, then i'm fine with merging

**Comment by TimeForANinja** [comment]
> That sounds fair, but I have no experience in testing, so here are a few questions:
> 
> 1. Should we add tests to `main-test` if we just export the method from `util`? If so, should we use the same tests in both of them or move the tests from `util-test` to `main-test`?
> 2. Should we use only callbacks/Promises in the tests, or are there any rules for this?
>    Thanks for your reply!

😉 guess you found out without my help
1) it does sometimes make sense to add a test whether it's correctly exportet (e.g. `util.xxx == main.xxx`) but i'm fine with just testing it in util
2) we should use promise once and callback once, for all other cases we can basically choose what we want

**Comment by TimeForANinja** [comment]
Thanks for the PR again :wink:

**Pull request merged**